e2e/terraform/gke/README.md

```diff
@@ -0,0 +1,39 @@
+# STEPS TO CREATE GKE CLUSTER USING TERRAFORM
+
+## Step-1 : Obtain the Google Credentials from Google Developers Console 
+
+- Log into the Google Developers Console and select a project.
+- The API Manager view should be selected, click on "Credentials" on the left, then "Create credentials," and finally "Service account key."
+- Select "Compute Engine default service account" in the "Service account" dropdown, and select "JSON" as the key type.
```

Comment:
Why `default` service account?

Reply:
It has the necessary permissions to create the cluster resources. We can use other sa with adequate permissions too (added a note in the README)

Reply:
Removed the placeholders to use TF_VAR environment variables.